apps/src/templates/codeDocs/ProgrammingExpressionOverview.jsx

```diff
@@ -11,6 +11,7 @@ import {
   shrinkBlockSpaceContainer
 } from '@cdo/apps/templates/instructions/utils';
 import {parseElement} from '@cdo/apps/xml';
+import '../../../style/curriculum/documentation_tables.scss';
```

Comment:
Yay thank you!! Does it also need to be added to the `ReferenceGuide` file?

Reply:
Yes.  Added.  I forgot that Reference Guides were a different file - I should have looked back at all the files changed.  I did a visual test on the ReferenceGuides since there was no test file.